### PR TITLE
Historical data page: Query only list .mcap file from S3 bucket and process request issues 

### PR DESCRIPTION
**Pull request by dan-du-car**
# PR Details
## Description

- Users can upload files if files do not exist in their own organization.
- ROS2 rosbag table should only display a list of .mcap file even though the S3 bucket has other types of files other than .mcap files.
- Preview table should show the browsed files that were immediately removed before.
- Abort upload request error should be display on UI.
- Process request is received by NATS server when uploading files to S3 bucket.

## Related GitHub Issue
https://github.com/usdot-fhwa-stol/cda-telematics/issues/178

## Related Jira Key
https://usdot-carma.atlassian.net/browse/WFD-513

## Motivation and Context
NA

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: telematic_system/telematic_apps/web_app/server/server.env [hunk 1]
```diff
@@ -33,6 +33,7 @@ UPLOAD_TIME_OUT=3600000 # Milliseconds
 UPLOAD_MAX_FILE_SIZE=21474836480 #20 GB
 CONCURRENT_QUEUE_SIZE=5 # How many parts can be parallel processed
 PART_SIZE=10485760 # The size of each part during a multipart upload, in bytes, at least 10MB
+FILE_EXTENSIONS=.mcap # Only query a list of objects with supported file extensions from S3 bucket
```
Comment (adev4a) [inline telematic_system/telematic_apps/web_app/server/server.env:36]: Same note as the .env files above

File: telematic_system/telematic.env [hunk 2]
```diff
@@ -79,6 +79,7 @@ UPLOAD_TIME_OUT=3600000 # Milliseconds
 UPLOAD_MAX_FILE_SIZE=21474836480 #20 GB
 CONCURRENT_QUEUE_SIZE=5 # How many parts can be parallel processed
 PART_SIZE=10485760 # The size of each part during a multipart upload, in bytes, at least 10MB
+FILE_EXTENSIONS=.mcap # Only query a list of objects with supported file extensions from S3 bucket
```
[the remark this answers is not in the document]
Comment (dan-du-car) [reply telematic_system/telematic.env:82]: It is a comma separated string

File: telematic_system/telematic.local.env [hunk 3]
```diff
@@ -51,6 +51,7 @@ UPLOAD_TIME_OUT=3600000 # Milliseconds
 UPLOAD_MAX_FILE_SIZE=21474836480 #20 GB
 CONCURRENT_QUEUE_SIZE=5 # How many parts can be parallel processed
 PART_SIZE=10485760 # The size of each part during a multipart upload, in bytes, at least 10MB
+FILE_EXTENSIONS=.mcap # Only query a list of objects with supported file extensions from S3 bucket
```
[the remark this answers is not in the document]
Comment (dan-du-car) [reply telematic_system/telematic.local.env:54]: It is a comma separated string

File: telematic_system/telematic_apps/web_app/server/server.env (hunk 1 above)
Comment (dan-du-car) [reply telematic_system/telematic_apps/web_app/server/server.env:36]: It is a comma separated string

**Review by adev4a — APPROVED** [review]

**Pull request merged**